app/lib/appframe.dart

```diff
@@ -39,10 +40,13 @@ class _AppFrameState extends State<AppFrame> {
         selectedIndex: _currentPageIndex,
         destinations: <Widget>[
           NavigationDestination(
-            selectedIcon: const Icon(Icons.home_outlined),
-            icon: const Icon(Icons.home),
-            label: AppLocalizations.of(context)!.frameHome,
-          ),
+              selectedIcon: const Icon(Icons.home_outlined),
+              icon: const Icon(Icons.home),
+              label: AppLocalizations.of(context)!.frameHome),
+          NavigationDestination(
+              selectedIcon: const Icon(Icons.map_outlined),
```

Comment:
Can we find a shorter name for 'Benachrichtigungen'? The navbar is getting crowded, and the text overflows and wraps to a second line

Reply:
Maybe 'News'?